libs/ui/elements/src/lib/external-link-card/external-link-card.component.ts

```diff
@@ -51,11 +51,21 @@ export class ExternalLinkCardComponent {
     return this.link.name || this.link.description || ''
   }
 
-  getLinkFormat(link: any) {
+  get isClickable(): boolean {
```

Comment:
As this is used for more than click (also the label and the icon), I would rename this to:
```suggestion
  get isPostgis(): boolean {
```
and same for all the "clickable/not clickable" mentions in this PR.

Reply:
I named it clickable so that it looks more generic and just not restricted to postgis (in future we can improve it).
It is currently used extra only for the table prefix.

Let me know if you still think naming it isPostgis is better.

Reply:
How about "isDatabase" to keep it generic then?